cmd/rp-rest/static/didcommvp.html

```diff
@@ -230,13 +230,20 @@ <h4 class="text-xl text-gray-900 leading-tight">Presented Digital ID</h4>
                 createClaimTable(flattened)
                 document.getElementById("proceedClick").href = '/creditsuccess';
 
+            } else if (csKey === "Credit Card Statement" && csKey === "Driving License") {
+                flattened = json['_claim_sources'][key]['claims']
+                replaceKeys(flattened)
+                console.log("flat here", flattened)
```

Comment:
Did you intend to keep this log statement in?

Reply:
I removed it,